model/annotations/task_annotations.go

```diff
@@ -125,6 +139,16 @@ func UpdateAnnotationNote(taskId string, execution int, originalMessage, newMess
 	return errors.Wrapf(err, "updating note for task '%s'", taskId)
 }
 
+// SetAnnotationMetadataLinks sets the metadata links for a task annotation.
+func SetAnnotationMetadataLinks(ctx context.Context, taskId string, execution int, metadataLinks ...MetadataLink) error {
+	query := ByTaskIdAndExecution(taskId, execution)
```

Comment:
Looking at the functions below, it seems like we may need to update source to be consistent with how annotations are used. cc @malikchaya2